add_sra_to_compendia.snakefile

```diff
@@ -0,0 +1,281 @@
+import pandas as pd
+import os
+
+m = pd.read_csv("inputs/metadata.csv", header = 0)
+m = m[m['accession_in_comp'] == False] # filter the metadata to public data that's not in the compendia
+m = m[m['pa_in_reads'] == True] # filter the metadata to samples that contained any Pa reads
+SRX = list(m['experiment_accession'])
+STRAIN = ['pao1', 'pa14']
+
+
+rule all:
+    input:
+        expand("outputs/multiqc/logs_{strain}_multiqc_report.html", strain = STRAIN),
```

Comment:
Are these files downloaded from somewhere? If they need to be downloaded and stored, this might need to be specified in the readme

Reply:
no! These are the target files that are generated by snakemake, so these are the products of the workflow. Snakemake works backwards from these file names to determine all of the rules that need to be run to generate these files.